Dockerfile

```diff
@@ -23,16 +23,20 @@ ARG GIT_SHA=0000000
 WORKDIR /src
 
 COPY pkg ./pkg
-COPY cmd ./cmd
 COPY go.mod ./
 COPY go.sum ./
 
 # Download all dependencies.
-RUN go mod download
+RUN go mod download -x
+
```

Comment:
Nit : To avoid bloating of console with download messages , avoid using `-x`

Reply:
Done